samples/bluetooth/fast_pair/locator_tag/sysbuild/mcuboot/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_BOOT_MAX_IMG_SECTORS=256
+
+# Ensure that the SPI NOR driver is disabled by default
+CONFIG_SPI_NOR=n
```

Comment:
Consider commenting that this is necessary to prevent build errors, because system clock is disabled in MCUboot's configuration